programs/drip/src/state/position.rs

```diff
@@ -21,8 +21,9 @@ pub struct Position {
     pub number_of_swaps: u64, // 8
     // deposit_amount_token_a / number_of_swaps
     pub periodic_drip_amount: u64, // 8
-    pub is_closed: bool,           // 1
-    pub bump: u8,                  // 1
+    // DEPRECATED FIELD: Position accounts are closed now instead of being marked closed
+    pub is_closed: bool, // 1
```

Comment:
Not quite sure how to test this without adding weird feature flags in the close positon ix to replicate the old behaviour. 

tldr:

We have a `close` function on the position, but after the anchor 0.29.0 upgrade, anchor itself exposes a `close` function on the account which takes precedence and actually closes the entire account. So our close function no longer gets called anywhere.